eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -387,52 +393,84 @@ function GetExistingTags($apiUrl) {
   }
 }
 
-# Walk across all build artifacts, check them against the appropriate repository, return a list of tags/releases
-function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $apiUrl, $releaseSha,  $continueOnError = $false) {
-  $pkgList = [array]@()
-  $ParsePkgInfoFn = ""
+# Retrieve release tag for artiface package. If multiple packages, then output the first one.
+function RetrieveReleaseTag($pkgRepository, $artifactLocation, $continueOnError = $true) {
+  if (!$artifactLocation) {
+    return ""
+  }
+  try {
+    $pkgs, $parsePkgInfoFn = RetrievePackages -pkgRepository $pkgRepository -artifactLocation $artifactLocation
+    if (!$pkgs -or !$pkgs[0]) {
+      Write-Host "No packages retrieved from artifact location."
+      return ""
+    }
+    if ($pkgs.Count -gt 1) {
+      Write-Host "There are more than 1 packages retieved from artifact location."
+      foreach ($pkg in $pkgs) {
+        Write-Host "The package name is $($pkg.BaseName)"
```

Comment:
Is BaseName the correct property? I don't see it defined in this file.

Reply:
This is System.IO.File property. Tested in command line.

Reply:
```
$a = Get-ChildItem ./
$a[0].BaseName
```

Reply:
`$pkgs` here is return var for `Get-ChildItem`

Reply:
OK I see I thought it was package info objects but that comes later.